var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -186,6 +186,13 @@ def install(self, spec, prefix):
 
         if '+shared' in spec:
             options.append('-DSHARED:BOOL=ON')
+        else:
+            rpath_options = []
```

Comment:
here and in `parmetis` could you please add a brief in-code comment on what is happening and why you need ot remove those RPATHs?